lessons/5_recap_and_practice/index.html

```diff
@@ -0,0 +1,231 @@
+<html>
+
+<head>
+	<title>Lesson 5</title>
+	<link rel="stylesheet" href="../../reveal.js/css/reveal.css">
+	<link rel="stylesheet" href="../../reveal.js/css/theme/moon.css">
+	<link rel="stylesheet" href="../../reveal.js/lib/css/zenburn.css">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/marked.js">
+	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/markdown.js">
+	<style>
+		b {
+			background-color: #ad0170;
+		}
+
+		.reveal pre {
+			margin: 0;
+		}
+
+		.monospace {
+			font-family: monospace !important;
+		}
+
+		.illustration {
+			border: 0px
+		}
+
+		.reveal li {
+			margin-bottom: 0.5em;
+		}
+
+		.reveal pre {
+			width: 100%;
+		}
+
+		.reveal pre code {
+			cursor: pointer;
+		}
+
+		.reveal li pre {
+			width: fit-content;
+		}
+
+		.logo {
+			position: fixed;
+			/* Sit on top of the page content */
+			z-index: 2;
+			/* Specify a stack order in case you're using a different order for other elements */
+		}
+	</style>
+</head>
+
+<body>
+	<a href="https://www.redi-school.org/"><img src="../../redi-school-logo.jpg" class="logo" /></a>
+	<div class="reveal">
+		<div class="slides">
+			<section>
+				<h1>Lecture</h1>
+				<h3>Recap and practice</h3>
+			</section>
+			<section>
+				<h3>What we would cover?</h3>
+				<ul>
+					<li class="fragment">General processes</li>
+					<li class="fragment">Homeworks structure</li>
+					<li class="fragment">mouseover/mouseleave events</li>
+					<li class="fragment">Practice</li>
+				</ul>
+			</section>
+			<section>
+				<h3>General processes</h3>
+				<h4 class="fragment">What does it mean?</h4>
+				<ul class="fragment">
+					<li>Homework rules</li>
+					<li>Gtihub process</li>
+					<li>Commitment</li>
+				</ul>
+			</section>
+			<section>
+				<h3>Homework rules</h3>
+				<ul>
+					<li>One homework per week (on Monday)</li>
+					<li>Deadline - Soft deadline on <b>Saturday 6PM</b>, hard <b>Sunday 12PM</b></li>
```

Comment:
Was the plan to have hard deadline just before next monday class?

Reply:
yes. hard <b>Sunday 12PM</b> is exactly that. Because I personally don't have time to check homework during working hours on Monday.